docs/source/examples/beam_output.txt

```diff
@@ -1,33 +1,29 @@
-     M : [ 1.8e+03   1.42e+03  1.09e+03  800      ... ]  [N*m] Internal moment     
-     V : [ 600       533       467       400      ... ]  [N]   Internal shear      
-\theta : [  -        0.0976    0.174     0.231    ... ]        Slope               
-     w : [  -        0.0325    0.123     0.258    ... ]  [m]   Displacement        
+    dx : 1                                               [m]   Length of an element
+     M : [ 1.25e+03  800       450       200      ... ]  [N*m] Internal moment     
+     V : [ 500       400       300       200      ... ]  [N]   Internal shear      
+\theta : [  -        0.103     0.165     0.198    ... ]        Slope               
+     w : [  -        0.0513    0.185     0.366    ... ]  [m]   Displacement        
 
 Constants
 ---------
-           EI : 1.1e+04                                         [N*m**2]                               
-            L : 6                                               [m]      Overall beam length           
-      M_{tip} : 0                                               [N*m]    Tip moment                    
-      V_{tip} : 0                                               [N]      Tip loading                   
-\theta_{base} : 0                                                        Base angle                    
-     w_{base} : 0                                               [m]      Base deflection               
-            M : [  -         -         -         -       ... ]  [N*m]    Internal moment               
-            V : [  -         -         -         -       ... ]  [N]      Internal shear                
-       \theta : [ 0          -         -         -       ... ]           Slope                         
-            q : [ 100       100       100       100      ... ]  [N/m]    Distributed load at each point
-            w : [ 0          -         -         -       ... ]  [m]      Displacement                  
+    EI : 1e+04                                           [N*m**2]                               
+     L : 5                                               [m]      Overall beam length           
+     M : [  -         -         -         -       ... ]  [N*m]    Internal moment               
+     V : [  -         -         -         -       ... ]  [N]      Internal shear                
+\theta : [ 0          -         -         -       ... ]           Slope                         
+     q : [ 100       100       100       100      ... ]  [N/m]    Distributed load at each point
+     w : [ 0          -         -         -       ... ]  [m]      Displacement                  
 
 Sensitivities
 -------------
  L : 4                                              Overall beam length           
- q : [ 0.0013    0.00762   0.0223    0.0454   ... ] Distributed load at each point
+ q : [ 0.0072    0.0416    0.118     0.234    ... ] Distributed load at each point
 EI : -1                                                                           
 
```

Comment:
Quick sanity checks: this output changed an awful lot for a simple discretization change from `N=10` to `N=6`. DO we know why? And why does the output have `L = 5` when `beam.py` sets L to 6?